src/Backend/Form/Type/MetaType.php

```diff
@@ -91,6 +113,48 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 ['label' => 'lbl.ExtraMetaTags', 'required' => false, 'attr' => ['rows' => 5, 'cols' => 62]]
             );
         }
+
+        $builder->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $formEvent) {
```

Comment:
Why do we need this?

Reply:
When an error occurs al fields stay empty. If use this these fields stay populated

Reply:
Not a fan at all of this solution, you switch back to working on arrays instead of on the entity or DTO, why don't you just update the values in the object?

Reply:
This happens before the data gets transformed in the ModelTransformer https://symfony.com/doc/current/form/events.html#a-the-formevents-pre-submit-event

The data returned is an array

Reply:
can you confirm that the constraints still work now that they are on the entity?

Reply:
@bjorvack do the constraints still work? I didn't see a confirmation here? Also still not approving of what you did here